src/sage/misc/lazy_attribute.pyi

```diff
@@ -0,0 +1,3 @@
+# This type-stub file helps pyright understand the decorator @lazy_attribute.
+
+lazy_attribute = property
```

Comment:
But this is not the correct type, right? lazy_attribute just happens to have the same typing as property (at the moment). In this case, just replace it by https://github.com/python/typeshed/blob/b9640005eb586afdbe0a57bac2b88a7a12465069/stdlib/builtins.pyi#L1237-L1254

Reply:
Thanks, done in 05e34ef